elements/bulbs-slick-slideshow/bulbs-slick-slideshow.js

```diff
@@ -19,6 +20,10 @@ class BulbsSlickSlideshow extends BulbsHTMLElement {
   attachedCallback () {
     this.slideshow = $('.slider');
 
+    if ($('.slider').length > 1) {
+      this.slideshow = $(this).find('.slider');
```

Comment:
golden 🔑

two scenarios: 
 - there's only one slideshow on the page, so `length` is 1 and that is the slideshow we want to use
 - a slideshow has already been initialized, so use `this` to find the one currently in view

for some reason the latter doesn't work for the former case, don't ask me why but this is the only consistent fix I could get across test/browser